setup.py

```diff
@@ -49,5 +49,15 @@
         "Programming Language :: Python",
         "Topic :: Internet",
         "Topic :: Software Development :: Libraries :: Python Modules",
-    ]
+    ],
+    entry_points={
+        'console_scripts': [
+            'eventlet_chat_bridge_example = examples.chat_bridge:__main__',
```

Comment:
Wait, installing eventlet also installs an importable `examples`?? That seems like a bug to me!

Reply:
Yeah that was more or less the idea, I used it locally to make tests and avoid creating patches based on copy of official examples. 

You are right, that's not the best idea that I had, let's abandon it.

It should have remained local hack :)